lib/screens/map/map_view_model.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:mobx/mobx.dart';
+
+import 'package:latlong2/latlong.dart' show LatLng;
+import 'package:flutter_map/flutter_map.dart';
+
+import 'package:api/dependency_injection.dart';
+
+import 'package:globo_fitness/screens/map/tree_marker.dart';
+
+import 'package:globo_fitness/store/tree_store.dart';
+
+import 'package:globo_fitness/template/view_model/view_model.dart';
+import 'package:globo_fitness/extensions/nullable_check.dart';
+
+part 'map_view_model.g.dart';
+
+class MapViewModel = MapViewModelBase with _$MapViewModel;
+
+abstract class MapViewModelBase with Store, ViewModel {
+  final TreeStoreBase treeStore =
+      DependecyInjection.instance.get<TreeStoreBase>();
+
+  final LatLng parisCoord = LatLng(48.866667, 2.333333);
+  final double defaultZoom = 13.0;
+  final double maxZoom = 18.0;
+  final int heightMarker = 50;
+  final int widthMarker = 50;
+  final int maxClusterRadius = 120;
+  final Size clusterSize = const Size(40, 40);
+
+  @observable
+  ObservableList<Marker> treesMarkers = ObservableList();
+
+  @override
+  void init() {
+    clearMarkerList();
```

Comment:
pourquoi un clear à l'init ?

Reply:
Currently it will add  the same markers again and have duplicates.
But I'll move the "clearMarkerList()" in the dispose method